libs/expo/shared/ui-components/src/lib/PillContainer/PillContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChevronLeftIcon } from '@monorepo/expo/shared/icons';
+import { Spacings } from '@monorepo/expo/shared/static';
+import { useState } from 'react';
+import { Pressable, StyleSheet, View } from 'react-native';
+import Pill from '../Pill';
+import TextBold from '../TextBold';
+
+export function PillContainer({
+  data,
+  type,
+}: {
+  data: string[];
+  type?: 'primary' | 'success';
+}) {
+  const [showAll, setShowAll] = useState(false);
+  const servicesToDisplay = showAll ? data : data.slice(0, 5);
```

Comment:
I'd move the `5` value out into a prop or constant. 
It's hard coded 3 times here and we should be able to switch this easily

maybe PillContainer could just accept a `maxVisible` prop.

Reply:
great idea. will update it

Reply:
done